Core/LocalAdmin.cs

```diff
@@ -789,6 +789,8 @@ private void RunScpsl()
             _processId = _gameProcess!.Id;
             Console.Title = BaseWindowTitle;
 
+            ConsoleUtil.WriteLine("Game process started with PID: " + _processId, ConsoleColor.DarkGreen);
```

Comment:
Wouldn't a quick check that the process started correctly be good here?

If the process ends up failing to start say, a corruption, how would this be handled?

Reply:
If the process fails to start then `.Start()` throws an exception that is handled by the catch (https://github.com/northwood-studios/LocalAdmin-V2/blob/master/Core/LocalAdmin.cs#L792).

And this is out of the scope of this PR.